app/test_main.py

```diff
@@ -1,3 +1,40 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years,dog_years,expected_result",
+    [
+        pytest.param(
+            14, 14, [0, 0],
+            id="return zeros if animal's years less than 15"
+        ),
+        pytest.param(
+            15, 15, [1, 1],
+            id="return 1 human's year if animal's years equal to 15"
+        ),
+        pytest.param(
+            23, 23, [1, 1],
+            id="return 1 human's year if animal's years less than 24"
+        ),
+        pytest.param(
+            24, 24, [2, 2],
+            id="return 2 human's year's if animal's years equal to 24"
```

Comment:
There is a typo in the description: 'year's' should be 'years'. It should read 'return 2 human's years if animal's years equal to 24'.